internal/jobservice/application.go

```diff
@@ -73,23 +83,30 @@ func (a *App) StartUp(ctx context.Context, config *configuration.JobServiceConfi
 			for _, value := range sqlJobRepo.GetSubscribedJobSets() {
 				log.Infof("Subscribed job sets : %s", value)
 				if sqlJobRepo.CheckToUnSubscribe(value.Queue, value.JobSet, config.SubscribeJobSetTime) {
-					sqlJobRepo.CleanupJobSetAndJobs(value.Queue, value.JobSet)
+					_, err := sqlJobRepo.CleanupJobSetAndJobs(value.Queue, value.JobSet)
+					if err != nil {
+						logging.WithStacktrace(log, err).Warn("error cleaning up jobs")
+					}
 				}
 			}
 		}
 		return nil
 	})
 	g.Go(func() error {
-		defer log.Infof("Stopping server.")
+		defer log.Infof("stopping server.")
 
-		log.Info("JobService service listening on ", config.GrpcPort)
+		log.Info("jobService service listening on ", config.GrpcPort)
```

Comment:
jobservice all lowercase?

Reply:
It would be two words so I think JobService or jobService would be correct?